### ISSUE TITLE
Improve init command

### ISSUE THREAD

**Issue by mitchellhamilton** [atlassian/changesets#160]
I'm thinking now that we have a JSON config, we could have the init command ask the user and write the config that they want.

The flow I'm thinking:

1. Do you want to generate changelogs? If so, which changelog entry generator would you like to use?
  - No, I don't want changelogs
  - Yes, I want changelogs with GitHub info (depends on #159)
  - Yes, I want changelogs with only commits
2. Would you like Changesets to commit changesets and version/changelog updates for you?
3. Should packages be published publicly or privately?

**Comment by Noviny** [comment]
This makes sense to me. I think #159 for common use-cases is probably higher priority though - wouldn't mind shipping some basic ones before making it obvious options

**Comment by mitchellhamilton** [comment]
Yes, 100%. I'm probably going to do #159 and then this issue after.
